src/View/HelperTrait.php

```diff
@@ -0,0 +1,37 @@
+<?php
+/**
+ * @see       https://github.com/zendframework/zend-i18n for the canonical source repository
+ * @copyright Copyright (c) 2005-2018 Zend Technologies USA Inc. (http://www.zend.com)
```

Comment:
Please use for new files only the current year. Thanks!

Reply:
I knew it! I was pretty sure it had to be just the current year (because that's how it was in the maintainer's repo), but was "blinded" by the other repositories, updated this year. Fixed that :+1: